src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -1050,7 +1062,12 @@
         }
       },
 
-      "required": ["fullyQualifiedLogicalName"]
+      "required": [ "fullyQualifiedLogicalName" ],
+      "dependencies": {
+        "uriBaseId": [ "uri" ],
```

Comment:
just playing devil's advocate but could the uriBaseId itself be sufficient? what does it look like if the very root of the uriBaseId is, for example, a scan target?